spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/domain/Trade.java

```diff
@@ -78,6 +84,13 @@ public static String dropDDL(String tableName) {
 		return "DROP table " + tableName;
 	}
 
+	private String getyyyymmddDate(Date date) {
```

Comment:
you can use `DateUtil.truncateTime()` from assertj instead of this method

Reply:
change made